packages/server/database/types/MeetingTeamPrompt.ts

```diff
@@ -33,7 +33,7 @@ export default class MeetingTeamPrompt extends Meeting {
   meetingType!: 'teamPrompt'
   meetingPrompt: string
   meetingSeriesId?: number
-  scheduledEndTime?: Date
+  scheduledEndTime?: Date | null | undefined
```

Comment:
+1 I think you can remove the `undefined` here, since it's already implied by `?`

Reply:
fixed